user.go

```diff
@@ -30,14 +30,14 @@ type UserConnection struct {
 }
 
 type UserIdentifierInput struct {
-	Id    *ID     `graphql:"id" json:"id,omitempty" defaults:"-"`
-	Email *string `graphql:"email" json:"email,omitempty" defaults:"-"`
+	Id    *ID     `graphql:"id" json:"id,omitempty" default:"-"`
```

Comment:
Blocking

Question -  can we just omit the `default` tag if there's no default to set, instead of setting it to `"-"`?

Reply:
Yes it looks like it behaves the same way